ecl/eclcc/eclcc.hpp

```diff
@@ -85,12 +85,12 @@ const char * const helpText[] = {
     "!   -brk <n>      Trigger a break point in eclcc after nth allocation",
 #endif
     "!   -Dname=value  Override the definition of a global attribute 'name'",
-    "!   --defaultgitprefix <prefix>  The default prefix used to access git repos when not specified in package.json",
+    "!?  --defaultgitprefix <prefix>  The default prefix used to access git repos when not specified in package.json",
```

Comment:
Should the order has to be "?!" for this to be included properly in ecl command usage (and for the markup character to be removed)?

Reply:
well spotted.